### PR TITLE
Rename s3 uri

### PR DESCRIPTION
**Pull request by tech3371**
# Change Summary
Rename to match database name:
* s3_uri to file_path
* file_to_create to file_to_create_path

### CODE REVIEW

File: sds_data_manager/lambda_code/SDSCode/batch_starter.py [hunk 1]
```diff
@@ -347,7 +345,7 @@ def send_lambda_put_event(command_parameters):
         "Source": "imap.lambda",
         "DetailType": "Job Started",
         "Detail": {
-            "file_to_create": "<s3_uri>",
+            "file_to_create_path": "<file_path>",
```
Comment (greglucas) [inline sds_data_manager/lambda_code/SDSCode/batch_starter.py]: I would put the path right after the file throughout here.
```suggestion
            "file_path_to_create": "<file_path>",
```

File: tests/lambda_endpoints/test_indexer_lambda.py [hunk 2]
```diff
@@ -132,6 +132,13 @@ def populate_db(test_engine):
         yield session
 
 
+@pytest.fixture()
+def set_env():
+    # Set the environment variable
+    os.environ["S3_DATA_BUCKET"] = "data-bucket"
+    return ""
```
Comment (greglucas) [inline tests/lambda_endpoints/test_indexer_lambda.py]: We should set this with setenv:
https://docs.pytest.org/en/latest/reference/reference.html#pytest.MonkeyPatch.setenv
Do you want to put this in a conftest.py file for all tests?

```suggestion
@pytest.fixture(autouse=True)
def set_env(monkeypatch):
    # Set the environment variable
    monkeypatch.setenv("S3_DATA_BUCKET", "test-data-bucket")
```

**Review by greglucas — APPROVED** [review]
Looks good to me. Just two minor comments.

File: sds_data_manager/lambda_code/SDSCode/batch_starter.py (hunk 1 above)
Comment (tech3371) [reply sds_data_manager/lambda_code/SDSCode/batch_starter.py]: I was trying to match what we called in the database. Should I rename all to that?

File: tests/lambda_endpoints/test_indexer_lambda.py (hunk 2 above)
Comment (tech3371) [reply tests/lambda_endpoints/test_indexer_lambda.py]: oh ok. Thank you!

File: sds_data_manager/lambda_code/SDSCode/batch_starter.py (hunk 1 above)
Comment (greglucas) [reply sds_data_manager/lambda_code/SDSCode/batch_starter.py]: My main comment is just that we're changing `file` -> `file_path`, not appending `path` to the variables. I think it sounds better that way at least.

So, IMO, yes I think all of the places that were updated to `file_to_create_path` should be renamed to `file_path_to_create`.

Comment (tech3371) [reply sds_data_manager/lambda_code/SDSCode/batch_starter.py]: changed that now.

**Pull request merged**